docs/feature-flags/sdks/ruby.md

```diff
@@ -57,16 +57,18 @@ client = EppoClient::init(config)
 
 The SDK will invoke the `log_assignment` function with an `assignment` object that contains the following fields:
 
-| Field                     | Description                                                                                                              | Example                  |
-| ------------------------- | ------------------------------------------------------------------------------------------------------------------------ | ------------------------ |
-| `experiment` (string)     | An Eppo experiment key                                                                                                   | "recommendation_algo"    |
-| `subject` (string)        | An identifier of the subject or user assigned to the experiment variation                                                | UUID                     |
-| `variation` (string)      | The experiment variation the subject was assigned to                                                                     | "control"                |
-| `timestamp` (string)      | The time when the subject was assigned to the variation                                                                  | 2021-06-22T17:35:12.000Z |
-| `subjectAttributes` (map) | A free-form map of metadata about the subject. These attributes are only logged if passed to the SDK assignment function | `{ "country": "US" }`    |
+| Field                     | Description                                                                                                              | Example                             |
+| ------------------------- | ------------------------------------------------------------------------------------------------------------------------ | ----------------------------------- |
+| `experiment` (string)     | An Eppo experiment key                                                                                                   | "recommendation-algo-allocation-17" |
+| `subject` (string)        | An identifier of the subject or user assigned to the experiment variation                                                | UUID                                |
+| `variation` (string)      | The experiment variation the subject was assigned to                                                                     | "control"                           |
+| `timestamp` (string)      | The time when the subject was assigned to the variation                                                                  | 2021-06-22T17:35:12.000Z            |
+| `subjectAttributes` (map) | A free-form map of metadata about the subject. These attributes are only logged if passed to the SDK assignment function | `{ "country": "US" }`               |
```

Comment:
Not your code, but I _think_ in the Ruby world people generally refer to these as "Hashes" ([docs](https://ruby-doc.org/core-2.7.1/Hash.html))